hawkularclient.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |gem|
   gem.homepage      = 'https://github.com/hawkular/hawkular-client-ruby'
   gem.summary       = 'A Ruby client for Hawkular'
   gem.license       = 'Apache-2.0'
-  gem.required_ruby_version = '>= 2.0.0'
+  gem.required_ruby_version = ['>= 2.2.0', '< 2.5']
```

Comment:
Good for updating from `2.0.0`, but I think we should avoid adding an upper-limit to the ruby version. New ruby versions very rarely break stuff, and this might break setups where the user might want to upgrade ruby without upgrading gem versions.

Reply:
Any reason why `2.5` is the upper limit?

Reply:
Just because 2.5 still does not exist. But I'll take @cfcosta suggestion and not place the upper bound.